.github/workflows/release.yml

```diff
@@ -0,0 +1,154 @@
+name: Release Web Media SDK
```

Comment:
The package.json is missing the semantic release packages.  Please include them all.

Reply:
updated